lib/src/_client.dart

```diff
@@ -359,6 +359,39 @@ class Client {
     );
   }
 
+  /// Discover terms in the index that match a partial String
+  Future<TermEnumResult> termEnum({
```

Comment:
Nit: Please add the API URL into the method dartdoc:
https://www.elastic.co/guide/en/elasticsearch/reference/8.1/search-terms-enum.html